src/znc.cpp

```diff
@@ -2121,3 +2152,10 @@ void CZNC::LeakConnectQueueTimer(CConnectQueueTimer* pTimer) {
 }
 
 bool CZNC::WaitForChildLock() { return m_pLockFile && m_pLockFile->ExLock(); }
+
+void CZNC::DisableConfigTimer() {
+  if (m_pConfigTimer) {
+    m_pConfigTimer->Stop();
+    m_pConfigTimer = nullptr;
```

Comment:
"manager"? CSockCommon::Cron automatically deletes timers when it detects them as being invalid.